src/go/rpk/pkg/cli/cmd/cluster/config/import.go

```diff
@@ -119,13 +128,16 @@ func importConfig(
 				upsert[k] = v
 			}
 		} else {
-			// Present in input but not original config, insert
-			upsert[k] = v
-			propertyDeltas = append(propertyDeltas, propertyDelta{k, "", fmt.Sprintf("%v", v)})
+			// Present in input but not original config, insert if it differs
+			// from the materialized current value (which may be a default)
+			if !haveOldValMaterialized || !reflect.DeepEqual(oldValMaterialized, v) {
+				upsert[k] = v
+				propertyDeltas = append(propertyDeltas, propertyDelta{k, fmt.Sprintf("%v", oldValMaterialized), fmt.Sprintf("%v", v)})
```

Comment:
Summary of this change,
This adds the old default to the delta, rather than `""`.

This changes the `PRIOR` output from empty when we change the old default, to now be the old default value.